scipy/linalg/_solvers.py

```diff
@@ -230,100 +230,221 @@ def solve_discrete_lyapunov(a, q, method=None):
     return x
 
 
-def solve_continuous_are(a, b, q, r):
+def solve_continuous_are(a, b, q, r, e=None, s=None, balanced=True):
     """
-    Solves the continuous algebraic Riccati equation (CARE).
+    Solves the continuous-time algebraic Riccati equation (CARE).
 
     The CARE is defined as
 
     .. math::
-        A'X + XA - XB R^{-1} B'X + Q = 0
 
-    It is solved directly using a Schur decomposition method.
+          XA + A'X - X B R^{-1} B'X + Q = 0
```

Comment:
Yes I wanted to do that but since the function works also with complex numbers I went on with prime notation. Maybe change to `^*` ?